nlp/notebook_config.yml

```diff
@@ -1,10 +1,15 @@
-# Config file containing the list of notebooks to download for a specific image.
+# Config file containing the list of notebooks directories to download for a specific image.
 #
 # Used in the deploy-data-specific-image script from pavics-jupyter-base.
 # More details on this config can be found on this script file :
 #   https://github.com/bird-house/pavics-jupyter-base/blob/master/scheduler-jobs/deploy_data_specific_image
 
 - repo_url: https://github.com/crim-ca/pavics-jupyter-images
   branch: master
-  source_path: nlp/notebooks
-  dest_sub_dir:
+  source_dir: nlp/notebooks
+  dest_sub_dir: nlp
+
+- repo_url: https://github.com/Ouranosinc/pavics-sdi
```

Comment:
Ouch double download of the pavics-sdi notebooks.  At least the config is clearer.

We could have one `config.yml` that will handle all the notebooks repos (eo, nlp, common) but then each image won't be "independent" anymore.

Don't have a quick solution at the moment.

Reply:
@tlvu, at least it's one copy per image and not per user. Hopefully, the common notebooks folder will stay small.

> We could have one config.yml that will handle all the notebooks repos (eo, nlp, common) but then each image won't be "independent" anymore.

That's the whole point of this PR! Having a single config.yml is about the same as what we have right now. 

The download script could be optimized (not for this PR) and keep a cache of each repo independently of the current image. So that while looping over the images, if a repo has already been updated, it uses the cache. 

But again I'm note sure of your concern, is it the download or the disk space? In both case it looks like trying to save pennies.

Reply:
Opps sorry, sending wrong impression again.  Wasn't saying this is a deal breaker, else I would change the approval status.

Probably the layout on disk could be
```
/data/jupyterhub_user_data/tutorial-notebooks/common
/data/jupyterhub_user_data/tutorial-notebooks/crim-eo
/data/jupyterhub_user_data/tutorial-notebooks/crim-nlp
```
Instead of
```
/data/jupyterhub_user_data/tutorial-notebooks/crim-eo/eo
/data/jupyterhub_user_data/tutorial-notebooks/crim-eo/common
/data/jupyterhub_user_data/tutorial-notebooks/crim-nlp/nlp
/data/jupyterhub_user_data/tutorial-notebooks/crim-nlp/common
```

But that also assume the same definition of `common` for those images, which might make sense only if on the same deployment node.  Like you said, it's pennies at this point so let's do this once we get there.

Reply:
Don't worry I'm not offended and sorry if this is the impression I leave.

Reply:
Just to be clear, I like all trade-offs/limitations to be clearly documented in a PR.  Doesn't mean we need to address all of them immediately.  I much prefer iterative approach where we improve gradually than keep adding feature to grow the PR and never release it!